Assets/MixedRealityToolkit/_Core/Extensions/CollectionsExtensions.cs

```diff
@@ -150,13 +150,30 @@ public static T[] ExportDictionaryValuesAsArray<T>(this Dictionary<uint,T> input
         /// <param name="input">The InteractionDefinition dictionary reference (generics, performed on a Dictionary)</param>
         /// <param name="key">The specific DeviceInputType value to update</param>
         public static bool GetDictionaryValueChanged(
-            this Dictionary<Definitions.Devices.DeviceInputType, Definitions.Devices.InteractionMapping> input,
-            Definitions.Devices.DeviceInputType key)
+            this Dictionary<Definitions.Devices.DeviceInputType, MixedRealityInteractionMapping> input, DeviceInputType key)
         {
             var entry = input[key];
             var changed = entry.Changed;
             input[key] = entry;
             return changed;
         }
+
+
+        /// <summary>
+        /// Overload extension to enable getting of an InteractionDefinition of a specific type
+        /// </summary>
+        /// <param name="input">The InteractionDefinition array reference</param>
+        /// <param name="key">The specific DeviceInputType value to query</param>
+        public static MixedRealityInteractionMapping GetInteractionbyType(this MixedRealityInteractionMapping[] input, DeviceInputType key)
```

Comment:
GetInteractionbyType -> GetInteractionByType?

Reply:
Fixed in a future PR I caught that too.